addon/components/ui-text-input/component.ts

```diff
@@ -21,10 +23,21 @@ export default class UiTextInput extends Component {
   disabled?: boolean = false;
   max?: number | null = null;
   min?: number | null = null;
+  width?: string; //Accepts any standard CSS width value
```

Comment:
Needs a default value;

Reply:
If it has a default value here, the template `if` would always be true, and if that's the case, we could just put the css straight into the file, couldn't we?

Reply:
Default can just be empty string "" but it needs to be a `string`, cant be `undefined`.